src/Behat/Behat/Context/Snippet/Generator/ContextSnippetGenerator.php

```diff
@@ -319,36 +278,27 @@ private function getMethodNameNotProposedEarlier($contextClass, $stepPattern, $n
     /**
      * Returns already proposed method names.
      *
-     * @param string $contextClass
-     *
-     * @return string[]
+     * @return array<string, array<string, string>
```

Comment:
This type is wrong. It does not return an array of arrays as it returns the proposed methods for a single context class. the proper return type is `array<string, string>`

Btw, the current annotation is even totally broken because it misses a `>` to be a valid type.

Reply:
Well spotted @stof, sorry I missed that. @fmatsos do you have time to fix that? Thanks

Reply:
Yes, I already have the fix on a new branch with the next proposal ;)

Reply:
Since the var_export question is resolved, I've just pushed a fix for this in #1569